evap/evaluation/tests/test_models.py

```diff
@@ -0,0 +1,29 @@
+import datetime
+from unittest.case import TestCase
+from unittest.mock import patch
+from model_mommy import mommy
+
+from evap.evaluation.models import Course
+
+
+class TestCourses(TestCase):
+    today = datetime.date.today()
+
+    def test_approved_course(self):
+        course = mommy.make(Course, state='approved',  vote_start_date=self.today)
+
+        with patch('evap.evaluation.models.EmailTemplate.send_evaluation_started_notifications') as mock:
+            Course.update_courses()
+
+        mock.assert_called_once_with([course])
+
+        course = Course.objects.get(pk=course.pk)
+        self.assertEqual(course.state, 'inEvaluation')
+
+    def test_fully_reviewed(self):
```

Comment:
please make all the test names descriptive enough to not need a description, or add a comment.